requirements.txt

```diff
@@ -7,3 +7,4 @@ pyDeprecate
 pandas>=1.1.0
 jsonargparse[signatures]>=3.17.0
 click>=7.1.2
+tqdm
```

Comment:
Do we need this? I think `tqdm` is included in PL requirements

Reply:
yeah I suppose we don't